ts/test/session/unit/models/formatRowOfConversation_test.ts

```diff
@@ -0,0 +1,297 @@
+        null,
+        'formatRowOfConversation with undefined should return null'
+      );
+    });
+  });
+
+  describe('groupAdmins', () => {
+    it('groupAdmins undefined fills it with []', () => {
+      expect(formatRowOfConversation({ groupAdmins: undefined })).to.be.have.deep.property(
+        'groupAdmins',
+        []
+      );
+    });
+
+    it('groupAdmins null fills it with []', () => {
+      expect(formatRowOfConversation({ groupAdmins: null })).to.be.have.deep.property(
+        'groupAdmins',
+        []
+      );
+    });
+
+    it('groupAdmins [] fills it with []', () => {
+      expect(formatRowOfConversation({ groupAdmins: '[]' })).to.be.have.deep.property(
+        'groupAdmins',
+        []
+      );
+    });
+
+    it('groupAdmins ["12345"] from db as string', () => {
+      expect(
+        formatRowOfConversation({ groupAdmins: '["12345"]' })
+      ).to.be.have.deep.property('groupAdmins', ['12345']);
+    });
+
+    it('groupAdmins ["12345", "52345"] fills it with []', () => {
+      expect(
+        formatRowOfConversation({ groupAdmins: '["12345", "52345"]' })
+      ).to.be.have.deep.property('groupAdmins', ['12345', '52345']);
+    });
+  });
+
+  describe('members', () => {
+    it('members undefined fills it with []', () => {
+      expect(formatRowOfConversation({ members: undefined })).to.be.have.deep.property(
+        'members',
+        []
+      );
+    });
+
+    it('members null fills it with []', () => {
+      expect(formatRowOfConversation({ members: null })).to.be.have.deep.property('members', []);
+    });
+
+    it('members [] fills it with []', () => {
+      expect(formatRowOfConversation({ members: '[]' })).to.be.have.deep.property('members', []);
+    });
+
+    it('members ["12345"] from db as string', () => {
+      expect(formatRowOfConversation({ members: '["12345"]' })).to.be.have.deep.property(
+        'members',
+        ['12345']
+      );
+    });
+
+    it('members ["12345", "52345"] fills it with []', () => {
+      expect(
+        formatRowOfConversation({ members: '["12345", "52345"]' })
+      ).to.be.have.deep.property('members', ['12345', '52345']);
+    });
+  });
+
+  describe('zombies', () => {
+    it('zombies undefined fills it with []', () => {
+      expect(formatRowOfConversation({ zombies: undefined })).to.be.have.deep.property(
+        'zombies',
+        []
+      );
+    });
+
+    it('zombies null fills it with []', () => {
+      expect(formatRowOfConversation({ zombies: null })).to.be.have.deep.property('zombies', []);
+    });
+
+    it('zombies [] fills it with []', () => {
+      expect(formatRowOfConversation({ zombies: '[]' })).to.be.have.deep.property('zombies', []);
+    });
+
+    it('zombies ["12345"] from db as string', () => {
+      expect(formatRowOfConversation({ zombies: '["12345"]' })).to.be.have.deep.property(
+        'zombies',
+        ['12345']
+      );
+    });
+
+    it('zombies ["12345", "52345"] fills it with []', () => {
```

Comment:
Should this not be `['12345', '52345']` instead of `[]`?

Reply:
yes

Reply:
👍